packages/twenty-front/src/modules/object-record/graphql/types/RecordGqlFields.ts

```diff
@@ -1 +1,3 @@
-export type RecordGqlFields = Record<string, any>;
+export type RecordGqlFields = {
+  [k: string]: boolean | RecordGqlFields | undefined;
```

Comment:
Since undefined should not be here, might be good to add a TODO and create an issue.

Reply:
As we discussed, we would like to have only `[key: string]: true | RecordGqlFields` to close the API and avoid having to handle undefined, key presence and false.

Reply:
Indeed would simplify DX !